### PR TITLE
readme: give an expectation for pace

### PR DESCRIPTION
**Pull request by pushcx**
Earlier today I attempted to upgrade Ruby on a project and was tragically crushed under the weight of new cops and expired. My ghost investigated standard but could not figure out whether avalanches of new cops were less common. I haunted @searls and he laid my questions to rest (alongside my cairn). I offer this PR to the project `README.md` in the hopes it is informative to other restless spirits.

### CODE REVIEW

**Comment by searls** [comment]
I mentioned this to @camilopayan and he said he'd take a look at the README as well, so connecting you here

**Comment by pushcx** [comment]
Cool! I see they're a regular contributor to the repo, so I expect their choice of wording would certainly be an improvement over a newcomer's rewrite of a tweet. I'm happy to help, but maybe this is best to just hand off?

File: README.md [hunk 1]
```diff
@@ -182,14 +182,17 @@ The beauty of Ruby Standard Style is that it's simple. No one wants to
...
-This gem saves you (and others!) time in three ways:
+This gem saves you (and others!) time in four ways:
 
 - **No configuration.** The easiest way to enforce consistent style in your
   project. Just drop it in.
 - **Automatically format code.** Just run `standardrb --fix` and say goodbye to
   messy or inconsistent code.
 - **Catch style issues & programmer errors early.** Save precious code review
   time by eliminating back-and-forth between reviewer & contributor.
+- **Deliberate pace.** We strive to take hassle of upgrading Rubocop out of each
```
Comment (foca) [inline README.md:193]:
```suggestion
- **Deliberate pace.** We strive to take the hassle of upgrading Rubocop out of each
```

I think this is more correct? But I might be wrong (English is not my first language), so feel free to ignore.

Comment (searls) [reply README.md]: Nope, you're right!

**Comment by searls** [comment]
Thanks!

**Pull request merged**